app/controllers/external_authors_controller.rb

```diff
@@ -76,7 +76,7 @@ def update
     end
     @invitation.mark_as_redeemed if @invitation && !params[:imported_stories].blank?
 
-    if @external_author.update_attributes(external_author_params[:external_author])
+    if @external_author.update_attributes(external_author_params[:external_author] || {})
```

Comment:
Rails previously returned an empty hash for nested params that were empty - now it returns nil. `update_attributes` will raise an error if it doesn't receive a hash.